frontend/js/leaderboard.js

```diff
@@ -102,33 +104,39 @@ class Leaderboard {
     const runScore = document.createElement('td');
     const runWordsTyped = document.createElement('td');
     const runWordsSeen = document.createElement('td');
-    const runDeleteButton = document.createElement('button');
 
     const bestRun = this.getUserBestRun(account.runs);
 
-    runDeleteButton.addEventListener('click', () => {
-      if (bestRun.id !== undefined){
-        this.deleteRun(bestRun.id);
-      } else {
-        console.log('Cannot delete null run.');
-        //TODO: maybe change to modal popup?
-      }
-    });
-
     userName.textContent = account.username;
     runScore.textContent = bestRun.score;
     runWordsTyped.textContent = bestRun.words_typed;
     runWordsSeen.textContent = bestRun.words_seen;
-    runDeleteButton.textContent = 'Delete';
-    runDeleteButton.classList.add("btn", "btn-primary", "link");
 
     runContainer.append(userName, runScore, runWordsTyped, runWordsSeen);
 
-    if (game.account && Number(game.account.id) === Number(account.id)) {
```

Comment:
I could be wrong in my assumption, but i had this logic here in case someone knew how to use dev tools. they could just delete the hidden class on the delete button of another user to show that button. might be wrong though.

either way-- nice refactor! 👍